src/styles/blocks/card.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
Agreed with previous comment